app/assets/stylesheets/components/_form.scss

```diff
@@ -0,0 +1,94 @@
+.form-wrap {
+    border-radius: 15px;
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+
+    
+    .btn, .actions .btn {
+        margin: 10px;
+        width: 30px;
+    }
+    
+    .form-container .actions .btn {
+        margin: 0;
+        width: 100%;
+    }
+    
+    .form-container {
+        background-color: darken($white, 3%);
+        border-radius: 15px;
+        justify-content: space-between;
+        margin: auto;
+        padding: 20px;
+    }
+    
+    .form-container .field-checkbox {
+        align-items: center;
+        display: flex;
+        margin-bottom: 16px;
+    }
+    
+    .form-container .field {
+        align-items: flex-start;
+        display: flex;
+        flex-direction: column;
+    }
+    
+    .form-container .field input {
+        margin: 10px 0 30px;
+        width: 85%;
+    }
+    
+    .form-container form {
+        display: flex;
+        flex-direction: column;
+        padding: 10px;
+    }
+    
+    .form-container h1 {
+        text-align: center;
+    }
+    
+    .form-container .btn-submit {
+        padding: 10px;
+        width: inherit;
+    }
+    
+    .form-container .btn {
+        margin: 10px;
+        width: inherit;
+    }
+
+    .form-container .links-container {
+      display: flex;
+    }
+
+    .form-container .links-container a:first-child {
+      margin-right: auto;
+    }
+
+    .form-container .links-container a {
+      font-size: 14px;
+      padding: 10px;
+      text-align: center;
+    }
+}
+
+@include desktop-lg {
+  .form-wrap {
+    .form-container {
+      width: 30%;
+    }
+
+    .form-container .field {
+      align-items: center;
+      align-items: flex-start;
```

Comment:
here only apply the flex-start, remove the first one.

Reply:
done